### ISSUE TITLE
not an issue - but a question about how to learn this Atom-Vim-Mode-Plus

### ISSUE THREAD

**Issue by wisdomlight** [t9md/atom-vim-mode-plus#1147]
I am new to Atom and to VimModePlus.
I am a little bit familiar with vim. Enough to do little harm.
I am finding it difficult to understand how to use VimModePlus. 
Just simple things like replace all occurances of a string from line 10 to line 20
is there a way to learn VimeModePlus??
Thank you

**Comment by t9md** [comment]
- Follow this [Tutorial](https://github.com/t9md/atom-vim-mode-plus/wiki/AdvancedTopicTutorial) to quickly grasp vmp.
- Read [this](https://github.com/t9md/atom-vim-mode-plus/wiki/DifferencesFromPureVim) to understand fundamental differences from pure Vim.
- Some minor or unknown features [here](https://github.com/t9md/atom-vim-mode-plus/wiki/YouDontKnowVimModePlus).
- Read description of each setting on "settings > packages > vim-mode-plus". I tried as explicit as possible for the purpose of each setting.
- Read [CHANGELOG](https://github.com/t9md/atom-vim-mode-plus/blob/master/CHANGELOG.md), it also explains explicitly with a tag such as FIX, NEW etc..

**Comment by wisdomlight** [comment]
@t9md 
Thank you for your reply.
I'd appreciate your help -
Is it possible to select a range of lines like in normal vim 10GV12G

**Comment by keevan** [comment]
> @t9md Thank you for your reply. I'd appreciate your help - Is it possible to select a range of lines like in normal vim 10GV12G

Yes you'd do just that, except it'd be using lowercase double `g` (if I'm understanding your example correctly) to select from (whole) lines 10-12.

So 10ggV12gg
